week1_intro/deep_crossentropy_method.ipynb

```diff
@@ -359,7 +361,7 @@
     "    ax.set_xlabel('position (x)')\n",
     "    ax.set_ylabel('velocity (v)')\n",
     "    \n",
-    "    # Sample a trajectory and draw it\n",
+    "    # Sample the trajectory and draw it\n",
```

Comment:
I'm reasonably sure that it should be "a trajectory", since we are sampling an arbitrary trajectory, not a specific one.